docker-compose.yml

```diff
@@ -76,8 +76,8 @@ services:
     extends:
       file: docker-compose.services.yml
       service: app
-    entrypoint: watchmedo auto-restart -d . -p '*.py'
-    command: -- celery worker -E -A inspirehep.celery -l INFO --purge --queues celery,migrator,indexer_task,matcher,assign
+    entrypoint: celery
```

Comment:
I think that in that case we can just skip entrypoint and leave the command as it was